ProjectNarrative.tex

```diff
@@ -111,27 +111,50 @@
+% rates to an estimated optical depth, which is often, but not necessarily,
+% a line integral through the simulation, neglecting multi-D or GR effects.
+% but this trivializes the transport to a simple ``lightbulb'' and critical effects in setting the radiation field are missed.
+where emission and absorption are treated locally by comparing reaction rates to approximate optical depths.
+However this misses any effects radiation transport may have on the matter field, such as gravitational redshift, scattering, and multidimensional effects.
 Recently, it has been shown that higher fidelity neutrino physics may alter the nucleosynthesis and dynamics in the collapsar \citep{miller:2020}.
 However, the extent to which how and where high fidelity neutrino transport impacts the outcomes of a collapsar is still relatively uncertain.
-A systematic landscape study, varying both collapsar parameters and neutrino transport fidelity, is needed to fully understand the impacts of neutirno transport fidelity on collapsar outcomes. 
-
-Using the new 3D general relativistic radiation magnetohydrodynamic (GRRMHD) code Pheobus currently in development at Los Alamos National Laboratory (LANL), I will  perform a suite of collapsar simulations with varying neutrino transport implementations.
-Pheobus currently includes several transport schemes from the simple leakage scheme to a highly accurate Monte Carlo method.
-This code is built upon the performance portable adaptive mesh refinement (AMR) software Parthenon \citep{grete:2022}, developed at LANL, and is highly performant.
-By incrementally varying the neutrino transport algorithms, these simulations will reveal systematic effects of the impact of neutirno transport on collapsar outcomes, focusing on nucleosynthesis and accretion disk dynamics.
+A systematic landscape study, varying both collapsar parameters and neutrino transport fidelity, is needed to fully understand the impacts of neutirno transport fidelity on collapsar outcomes.
+
+% JMM: One thing you don't talk about here is that stellar parameters
+% matter quite a lot, and possibly swamp everything else. A big
+% challenge with Pheobus will be to vary stellar structure and initial
+% conditions. Rather than starting with a disk, we'll want to see if
+% we can make one form dynamically.
+
+Using the new 3D general relativistic radiation magnetohydrodynamic (GRRMHD) code Pheobus currently in development at Los Alamos National Laboratory (LANL), I will  perform a suite of collapsar simulations with varying neutrino transport methods and physical parameters.
+% JMM: Phoebus doesn't currently contain a leakage scheme, though it wouldn't be hard to implement one.
+Pheobus currently includes several transport schemes, including a moment method, a sophisticated Monte Carlo scheme \citep{nubhlight}, and a new hybrid method capable of spanning the range of possible optical depths \citep{MOCMC}.
```

Comment:
Actually, now that I wrote this, I realize this is wrong. Phoebus has thin cooling, which is the zero optical depth limit of Leakage.

Reply:
:point_up: fixed in most recent commit